mail_debrand/models/mail_template.py

```diff
@@ -49,3 +54,15 @@ def _debrand_body(self, html):
     def render_post_process(self, html):
         html = super().render_post_process(html)
         return self._debrand_body(html)
+
+    def _get_template_lang(self):
+        template_lang = False
+        if self._context.get("default_template_id"):
+            template = self.env["mail.template"].browse(
+                self._context.get("default_template_id")
+            )
+            if template.lang and self._context.get("active_id"):
+                template_lang = template._render_template(
```

Comment:
The `_render_template()` does not return a lang, are you sure this is correct? Can you write a test to cover this part?

Reply:
There are several example of this in the standard Odoo code:

like this one in mail.thread:

`template = self.env['mail.template'].browse(self.env.context['default_template_id'])
            if template and template.lang:
                lang = template._render_template(template.lang, self.env.context['default_model'], self.env.context['default_res_id'])`

or this one in sale.order:

`template = self.env['mail.template'].browse(template_id)
        if template.lang:
            lang = template._render_template(template.lang, 'sale.order', self.ids[0])`

Reply:
Could you write a test to cover this part?